### PR TITLE
Make all Hadoop related IOs use the same version property

### PR DESCRIPTION
**Pull request by iemejia**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`.
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by iemejia** [comment]
R: @dhalperi 
CC: @jbonofre

File: sdks/java/io/pom.xml [hunk 1]
```diff
@@ -60,6 +60,28 @@
...
+        <version>${hadoop.version}</version>
+        <scope>test</scope>
+      </dependency>
+
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-hdfs</artifactId>
+        <version>${hadoop.version}</version>
+        <classifier>tests</classifier>
+        <scope>test</scope>
```
Comment (dhalperi) [inline sdks/java/io/pom.xml]: LGTM as is as an improvement over status quo.

But, remind me why this isn't all at the top level?

Comment (iemejia) [inline sdks/java/io/pom.xml]: You definitely have a point, I think this evolved organically from the 'old' HdfsIO days until this point. I will move them to the parent pom.

**Comment by iemejia** [comment]
Retest this please

**Comment by jbonofre** [comment]
R: @jbonofre

**Comment by jbonofre** [comment]
Can you please rebase (to fix the conflict) ?

**Comment by iemejia** [comment]
Retest this please

**Comment by jbonofre** [comment]
Thanks, I'm taking a new look and waiting the end of Jenkins.

**Comment by jbonofre** [comment]
Let me check why Jenkins fails (by the way, I'm fixing an issue on SpannerIOIT).

**Comment by jbonofre** [comment]
Can you rebase and fix the conflict in order to do a new Jenkins run ? Thanks !

**Comment by iemejia** [comment]
Thanks, please take a look also at my other PR the one on JDK 9, it is quite simple and it would be nice if it also gets included for 2.1.0.

**Pull request closed without merging**